x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -391,6 +395,35 @@ class PackagePolicyService {
 
     return Promise.all(inputsPromises);
   }
+
+  public async runExternalCallbacks(
```

Comment:
This function signature is different from most in Fleet. We generally aim to use 3 or fewer params and use an object for many params or when order isn't obvious.

It's not a blocker and, if not for `schema`, would be the order of the `ExternalCallback` so it's not an arbitrary order. 

We can leave it as-is and come back to it later.